bluebottle/events/views.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework_json_api.views import AutoPrefetchMixin
+
+from bluebottle.activities.permissions import ActivityPermission
+from bluebottle.events.models import Event, Participant
+from bluebottle.events.serializers import EventSerializer, ParticipantSerializer
+from bluebottle.utils.permissions import (
+    OneOf, ResourcePermission, ResourceOwnerPermission)
+from bluebottle.utils.views import RetrieveUpdateAPIView, ListCreateAPIView, JsonApiViewMixin
+
+
+class EventList(JsonApiViewMixin, AutoPrefetchMixin, ListCreateAPIView):
```

Comment:
I think we should use a Polymorphic activity  list endpoint instead of an endpoint specific for the activity type.

Reply:
Possibly we do need an create endpoint for every activity type, because ember does not really like polymorphic endpoints that return a single resource.